packages/api/tests/lambdas/test-kinesis-consumer.js

```diff
@@ -3,23 +3,32 @@
 const get = require('lodash/get');
 const sinon = require('sinon');
 const test = require('ava');
+const proxyquire = require('proxyquire');
 
 const { randomString } = require('@cumulus/common/test-utils');
-const SQS = require('@cumulus/aws-client/SQS');
 const { s3, sns } = require('@cumulus/aws-client/services');
 const { recursivelyDeleteS3Bucket } = require('@cumulus/aws-client/S3');
-const { handler } = require('../../lambdas/message-consumer');
+
 const Collection = require('../../models/collections');
 const Rule = require('../../models/rules');
 const Provider = require('../../models/providers');
 
+const sandbox = sinon.createSandbox();
+const queueMessageStub = sandbox.stub().resolves(true);
```

Comment:
Because in some of the tests we do actually check the stub to ensure that things are getting called correctly:

https://github.com/nasa/cumulus/pull/1782/files?file-filters%5B%5D=.js#diff-b09b95152f67f433a1a926062c3e34adR163

Reply:
Ah, missed the callcount checks.    👍